src/ILLink.Tasks/ILLink.Tasks.csproj

```diff
@@ -42,8 +42,23 @@
       <SetTargetFramework>TargetFramework=netstandard2.0</SetTargetFramework>
     </ProjectReference>
     <ProjectReference Condition="('$(UseCecilPackage)' != 'true') And ('$(TargetFramework)' != 'net472')" Include="../../external/cecil/symbols/pdb/Mono.Cecil.Pdb.csproj" PrivateAssets="All" />
+    <ProjectReference Include="../ILLink.RoslynAnalyzer/ILLink.RoslynAnalyzer.csproj">
```

Comment:
It might be worth conditioning the ProjectReference on the tfm so the assemblies only get copied over to one output path.